gradio/blocks.py

```diff
@@ -12,7 +12,7 @@
 import webbrowser
 from abc import abstractmethod
 from types import ModuleType
-from typing import TYPE_CHECKING, Any, AsyncIterator, Callable
+from typing import TYPE_CHECKING, Any, AsyncIterator, Callable, Literal
```

Comment:
We also use `typing_extensions` a few times in the python client. Would be good to get rid of those too, as well as the dependency`typing_extensions` altogether from the client

Reply:
We need typing_extensions for `NotRequired` still!

Reply:
Gotcha. Looking forward to Gradio being Python 3.11+!